tests/unit/components/sl-tooltip-test.js

```diff
@@ -8,7 +8,77 @@ moduleForComponent( 'sl-tooltip', 'Unit | Component | sl tooltip', {
 
 test( 'Expected Mixin is present', function( assert ) {
     assert.ok(
-        TooltipEnabledMixin.detect( this.subject() ),
+        TooltipEnabledMixin.detect( this.subject( { title: 'Tooltip Text' } ) ),
         'Expected Mixin is present'
     );
 });
+
+test( 'enabledTooltip() - Verify classes are set correctly: data-toggle and data-original-title', function( assert ) {
```

Comment:
The name of the test does not need to be so specific (the inclusion of the specific class names).  This is so that there are less things to be out of sync with the code if a class is added or removed in the future.